.github/workflows/build_release_candidate.yml

```diff
@@ -245,8 +245,7 @@ jobs:
 
   stage_docker:
     if:  ${{ fromJson(github.event.inputs.STAGE).docker_artifacts == 'yes'}}
-    # Note: if this ever changes to self-hosted, remove the "Remove default github maven configuration" step
```

Comment:
Note: I tried removing this and it caused issues, so I think this comment is wrong